.github/workflows/deb.yml

```diff
@@ -0,0 +1,77 @@
+name: Build Debian Packages
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Version to release'
+        required: true
+      upload-artifacts:
+        description: 'Upload artifacts to PackageCloud'
+        required: false
+        default: false
+      
+jobs:
+  build-deb:
+    runs-on: ${{ matrix.runner }}
+    container:
+      image: timescaledev/rust-pgx:latest
+    strategy:
+      matrix:
+        pgversion: [12, 13, 14]
+        runner:
+          - ubuntu-18.04
+          - ubuntu-20.04
+          - ubuntu-22.04
+    env:
+      OUTDIR: target/release/timescaledb_toolkit-pg${{ pgversion }}
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.inputs.version }}
+
+      - name: Cache cargo directories
+        uses: actions/cache@v2
+        with:
+          path: |
+            /usr/local/cargo/registry
+            /usr/local/cargo/git
+          key: ${{ runner.os }}-test-pg${{ matrix.pgversion }}-cargo-1.8
+          restore-keys: ${{ runner.os }}-test-pg${{ matrix.pgversion }}-cargo-1.5
+
+      - name: Cache cargo target dir
+        uses: actions/cache@v2
+        with:
+          path: target
+          key: ${{ runner.os }}-test-pg${{ matrix.pgversion }}-target-1.8
+          restore-keys: ${{ runner.os }}-test-pg${{ matrix.pgversion }}-target-1.5
+
+      - name: Build the package files
+        run: |
+          cargo pgx package --manifest-path=extension/Cargo.toml --out-dir=$OUTDIR
+          cargo run --manifest-path=tools/post-install/Cargo.toml -- pg_config --dir=$OUTDIR
+
+      - name: Build Debian Package
+        id: package
+        uses: timescale/build-actions/build-deb@v1
```

Comment:
Where can we see what happens here?  We want to test these debs before publishing, as documented https://docs.google.com/document/d/1P5SWP8OZy4Yx6-ddXSQ7EYNLrkrBjv-Z94-IO-2AYdg/edit

Will the RPM approach look very similar?

Reply:
The RPM approach will look similar.

It is still in draft, but the intention is that it should be possible to build the packages without releasing them and test them separately.